source/docs/steeltoe-discovery.html.markdown

```diff
@@ -5,62 +5,72 @@ date: 2018/2/8
 
-There are several options to choose from when implementing the Service Discovery pattern. Steeltoe has initially chosen to support one based on Eureka; using Netflix's Service Discovery server and client. For more information about Eureka see the [Netflix/Eureka Wiki](https://github.com/Netflix/eureka/wiki) and the [Spring Cloud Netflix](http://projects.spring.io/spring-cloud/) documentation.
-
-In the future you can expect to see more Service Discovery options as part of the Steeltoe framework.
+There are several options to choose from when implementing the Service Discovery pattern. Steeltoe has initially chosen to support one based on Eureka and using Netflix's Service Discovery server and client. For more information about Eureka, see the [Netflix/Eureka Wiki](https://github.com/Netflix/eureka/wiki) and the [Spring Cloud Netflix](http://projects.spring.io/spring-cloud/) documentation.
 
 # 0.0 Initialize Dev Environment
 
-All of the Steeltoe sample applications are in the same repository. If you haven't already, use git to clone the repository or download with your browser from GitHub: [Steeltoe Samples](https://github.com/SteeltoeOSS/Samples)
+All of the Steeltoe sample applications are in the same repository. If you have not already done so, use git to clone the [Steeltoe Samples](https://github.com/SteeltoeOSS/Samples) repository or download it with your browser from GitHub. The following example shows how to clone it with git:
 
 ```bash
 > git clone https://github.com/SteeltoeOSS/Samples.git
 ```
 
-> Note: all Service Discovery samples in the Samples repository have a base path of `Samples/Discovery/src/`
+>Note: All Service Discovery samples in the Samples repository have a base path of `Samples/Discovery/src/`.
 
-Make sure your Cloud Foundry CLI tools are logged in and targeting the correct org and space:
+Make sure your Cloud Foundry CLI tools are logged in and targeting the correct org and space, as follows:
 
 ```bash
 > cf login [-a API_URL] [-u USERNAME] [-p PASSWORD] [-o ORG] [-s SPACE] [--skip-ssl-validation]
-or
+```
+
+Alternatively, you can use the following command:
+
+```bash
 > cf target -o <YourOrg> -s <YourSpace>
 ```
 
 The Service Discovery sample requires a Eureka server. If you intend to run the samples locally, install the Java 8 JDK and Maven 3.x now.
 
 # 1.0 Netflix Eureka
 
-The Steeltoe Eureka client enables applications to register services with a Eureka server and to discover services registered by other applications. This Steeltoe client is an implementation of the 1.0 version of the Netflix Eureka client.
+The Steeltoe Eureka client lets applications register services with a Eureka server and discover services registered by other applications. This Steeltoe client is an implementation of the 1.0 version of the Netflix Eureka client.
```

Comment:
@jkonicki do we have branding concerns here with regard to "Steeltoe Eureka" ?

Reply:
@TimHess Yes, same concern.  Renaming to "Steeltoe's Eureka client implementation" would be better.